lib/fx/Clouds.sc

```diff
@@ -0,0 +1,19 @@
+CloudsPedal : Pedal {
+  *id { ^\clouds; }
+
+  *fxArguments { ^[\pit, \pos, \size, \dens, \tex, \spread, \mode, \fb ]; }
+
+  *fxDef {^{|wet|
+    var pit=0, pos=0.5, size=0.5, dens=0.5, tex=0, drywet=1, in_gain=1, spread=0.5, rvb=0.5, fb=0, freeze=0, mode=0, lofi=0, trig=0;
+	pit = \pit.kr(0.7);
```

Comment:
some of these defaults are a little confusing to me -- e.g. why does pitch start at `0.7`?

Reply:
I probably don't know what I'm doing. Is that a starting value or a default value? (are those different things?)

I was thinking they were just some defaults, so I threw some from some of the helpfile examples.